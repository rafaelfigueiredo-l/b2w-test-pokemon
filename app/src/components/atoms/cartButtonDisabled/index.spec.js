import CartButtonDisabled from "./";

describe("<CartButtonDisabled />", () => {
  const image = require("../../../assets/images/pokemonImageNotFound.png");

  const component = <CartButtonDisabled>Texto</CartButtonDisabled>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartButtonDisabled component", () => {
    expect(wrapper.find(".cartButtonDisabled")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartButtonDisabled component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartButtonDisabled").children()).toHaveLength(1);
  });
});
