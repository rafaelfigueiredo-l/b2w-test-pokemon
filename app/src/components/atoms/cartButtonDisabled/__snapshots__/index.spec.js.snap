// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartButtonDisabled /> should mount correctly CartButtonDisabled component 1`] = `
<styled.button
  className="cartButtonDisabled"
>
  <button
    className="sc-AxjAm bEQOOo cartButtonDisabled"
  >
    Texto
  </button>
</styled.button>
`;
